docs/components/ListView.jsx

```diff
@@ -0,0 +1,231 @@
+import _ from "lodash";
+import React, {Component} from "react";
+import {Link} from "react-router";
+
+import Example, {CodeSample, ExampleCode} from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import {List, FlexBox, ItemAlign, SegmentedControl} from "src";
+
+import "./ListView.less";
+
+const Items = {
+  empty: [],
+  nonEmpty: [
+    "Raindrops on roses",
+    "Whiskers on kittens",
+    "Bright copper kettles",
+    "Warm woolen mittens",
+    "Cream-colored ponies",
+    "Crisp apple strudels",
+    "Door bells",
+    "Sleigh bells",
+    "Schnitzel with noodles",
+  ],
+};
+
+export default class ListView extends Component {
+  static cssClass = {
+    CONFIG_CONTAINER: "ListView--configContainer",
+    CONFIG_OPTIONS: "ListView--configOptions",
+    CONFIG: "ListView--config",
+    CONTAINER: "ListView",
+    INTRO: "ListView--intro",
+    LIST: "ListView--list",
+    PROPS: "ListView--props",
+  };
+
+  state = {
+    itemsKey: "nonEmpty",
+    rowType: List.defaultProps.rowType,
+    showCustomEmptyMessage: false,
+    showTitle: true,
+    showBorder: true,
+  };
+
+  componentWillMount() {
```

Comment:
Why is this needed? The entire component lifecycle is invoked even without an empty function, right?